### PR TITLE
Update Docker page.

### PR DESCRIPTION
**Pull request by d7415**

### CODE REVIEW

File: docs/md/docker.md [hunk 1]
```diff
@@ -63,39 +57,31 @@ alter user 'lychee' identified with mysql_native_password by '<your password>';
...
 
-Note that nginx will accept by default images up to 100MB (`client_max_body_size 100M`) and that PHP parameters are overridden according to the [recommendations of the FAQ](faq.html#i-cant-upload-photos).
+Note that nginx will accept by default images up to 100MB (`client_max_body_size 100M`) and that PHP parameters are overridden according to the [recommendations of the Lychee FAQ](https://lycheeorg.github.io/docs/faq.html#i-cant-upload-photos).
 
 You may still want to further customize PHP configuration. The first method is to mount a custom `php.ini` to `/etc/php/7.3/fpm/php.ini` when starting the container. However, this method is kind of brutal as it will override all parameters.
 
 Instead, we recommend to use the `PHP_VALUE` directive of PHP-FPM to override specific parameters. To do so, you will need to mount a custom `nginx.conf` in your container :
 
-* 1 &mdash; Take the [default.conf](https://github.com/LycheeOrg/Lychee-Laravel-Docker/default.conf) file as a base
+* 1 &mdash; Take the [default.conf](./default.conf) file as a base
```
Comment (ildyria) [inline docs/md/docker.md:84]: you sure on that one ?

File: docs/md/docker.md [hunk 2]
```diff
@@ -63,39 +57,31 @@ alter user 'lychee' identified with mysql_native_password by '<your password>';
 
 ### Run with Docker Compose
 
-Change the environment variables in the [provided example](https://github.com/LycheeOrg/Lychee-Laravel-Docker/docker-compose.yml) to reflect your database credentials.
+Change the environment variables in the [provided example](./docker-compose.yml) to reflect your database credentials.
```
Comment (ildyria) [inline docs/md/docker.md:60]: you sure on that one ?

File: docs/md/docker.md (hunk 1 above)
Comment (d7415) [reply docs/md/docker.md:84]: Good spot!

Comment (d7415) [reply docs/md/docker.md:84]: Those were like that before - you must have changed them for v1 of the docs. Those aren't mine lol

**Pull request merged**